src/org/opensolaris/opengrok/util/Executor.java

```diff
@@ -234,19 +258,21 @@ public void run() {
                         "Waiting for process interrupted: " + cmdList.get(0), e);
             }
         } finally {
+            // Stop timer thread if the instance exists.
+            if (timer != null) {
+                timer.cancel();
+            }
             try {
                 if (process != null) {
+                    closeStreams(process);
                     ret = process.exitValue();
                 }
             } catch (IllegalThreadStateException e) {
                 if (process != null) {
+                    closeStreams(process);
                     process.destroy();
```

Comment:
`process` will be destroyed twice. Is it supported? JavaDoc does not say...

Reply:
this was an oversight, removed.